src/CsFixerConfig.php

```diff
@@ -117,16 +117,18 @@ public static function create(): ConfigInterface
     /**
      * @param array<string, mixed> $rules
      */
-    public function addRules(array $rules): ConfigInterface
+    public function addRules(array $rules): self
```

Comment:
Here, we can add this annotation to make it visible that this method uses a fluent interface:

```php
@return $this
```